### ISSUE TITLE
Download + contact links broken

### ISSUE THREAD

**Issue by NoelLH** [koken/issues#62]
I've just encountered koken.me for the first time and was keen to try the standalone download and have a look. But it looks like help.koken.me links (including download and contact) are currently broken:

<img width="781" alt="Screenshot 2019-08-20 at 08 18 09" src="https://user-images.githubusercontent.com/3274454/63326112-707ca780-c323-11e9-9465-de9203e25cee.png">

**Comment by donniefitz2** [comment]
It's sad but the current owners have apparently abandoned it. They haven't been reachable for about 2 years.

**Comment by nickian** [comment]
This project is dead now? At least release it to GitHub. Maybe others can maintain it.

**Comment by nickian** [comment]
Well, I found this after some Googling: https://s3.amazonaws.com/koken-installer/releases/Koken_Installer.zip

**Comment by donniefitz2** [comment]
Unfortunately that's just the initial installer script. It won't run because the server behind that is gone. You can find Koken, a full install and all the plugins on GitHub. I can't remember where, but it's here somewhere.

**Comment by nickian** [comment]
No, if you try it, it works. All of the files referenced in the `index.php` file are on AWS. It still installs fine with the installer for me.

**Comment by nickian** [comment]
I wonder what the status of the license is?: http://koken.me/eula.html

I wish they would update it to be MIT or something like that with no restrictions.

**Comment by donniefitz2** [comment]
Oh, wow. I think I tried a few months back and it didn't work. They must be paying the AWS bill still ;)

**Comment by nickian** [comment]
Apparently. I would like to pull the files off to my own server and update the installer accordingly. I don’t know if this is something the original developer would approve of or not.

**Comment by donniefitz2** [comment]
I think you're probably okay as long as you don't re-distribute it. But, I'm no lawyer.

**Comment by nickian** [comment]
Well, here, I consolidated the install files so there is no external server dependency and modified the install script to work accordingly: https://github.com/nickian/koken-installer

**Comment by donniefitz2** [comment]
Nice work. I'm going to give it a try soon. Also, I did a lot of research on the company that owns Koken. Looks like they abandoned their primary product as well. I tried calling them and left a message a few weeks ago and never heard back. Not sure if there's much hope of getting in contact to discuss the license.
